java-checks-test-sources/src/main/java/checks/TooManyMethodsCheck.java

```diff
@@ -48,15 +57,15 @@ public void method10() {}
   };
 
   TooManyMethodsCheckC c2 = new TooManyMethodsCheckC() { // compliant : only overriden methods
-    public void method1() {}
-    public void method2() {}
-    public void method3() {}
-    public void method4() {}
-    public void method5() {}
+    @Override public void method1() {}
+    @Override public void method2() {}
+    @Override public void method3() {}
+    @Override public void method4() {}
+    @Override public void method5() {}
```

Comment:
It seems that the annotation was omitted on purpose, to test that the override is detected without the help of the annotation. We should not add them, or add an equivalent test without.

Reply:
i'll remove some of them